### PR TITLE
Update blobs to KRT16S.

### PR DESCRIPTION
**Pull request by maniac103**
Change-Id: Idf2c7ade503251c35e306bfa7af93637b96c6861

### CODE REVIEW

**Comment by ghost** [comment]
I don't know which pull request to follow https://github.com/TheMuppets/proprietary_vendor_lge/pull/24 or https://github.com/TheMuppets/proprietary_vendor_lge/pull/21

but several people on XDA in some unofficial cm-11.0 ROMs are working with what's presently been committed for the KRT16S blobs and they're having issues.

http://forum.xda-developers.com/showthread.php?t=2139784

When they get something to compile successfully something falls short.  It's a piecemeal of still using the older JB .84 baseband versus the new KRT .97 baseband but Wi-Fi/in-call audio/among other things don't work consistently atm.

TLDR- latest mako binary blobs in the mako folder for cm-11.0 don't work with KRT16S baseband M9615A-CEFWMAZM-2.0.1700.97 yet ... from what I can tell.

regardless, it's a good thread to follow for now if someone's wanting to get some temporary feedback on mako cm-11.0 atm :)

**Comment by Haifen** [comment]
@jawz101 What about commits ead1f7e90fd1b5ccaa4a20b36ad2e10d77f9805f and 8139f088af6a0a7ef0aacfb16c4f1b849daeb4bd?

**Comment by ghost** [comment]
I'm not sure I understand the question.  You mean, what commits am I using presently?  I'll have to try again when I get off of work.  
I was up to date as of ead1f7e90fd1b5ccaa4a20b36ad2e10d77f9805f
I haven't sync'd to get this one yet 8139f088af6a0a7ef0aacfb16c4f1b849daeb4bd

**Comment by ghost** [comment]
I'll get you better information in a few hours once I leave the office and get back in front of my home computer.
